src/quest_manager/tests/test_views.py

```diff
@@ -265,6 +266,26 @@ def test_student_no_quests_help_text(self):
         self.assertContains(response, "No quests are available.")
         self.assertContains(response, "There are currently no new quest available to you!")
 
+    def test_QuestCopy__correct_clone(self):
```

Comment:
No docstring... what is this testing?

Reply:
turns out theres a whole TestClass dedicated to quest copy